### ISSUE TITLE
Question PCB

### ISSUE THREAD

**Issue by F0EOS** [f8kgl/swr_power_meter#407]
Pour J4 et J7 ne peut on pas mettre des points info comme J2 , J3 ou J6 
si je me trompe pas c'est la que vas venir "la tension" mesurée aux bornes de la ligne coaxial 

pour j4 et j7 je peut on pas les mettre de chaque cote du PCB  toujours par rapport a la ligne coaxiale ??

ne doit on pas fixer une taille maxi du PCB ?? afin d'optimiser au mieux l'emplacement des composants ? peut être a voir en fonction de la ligne coaxial si elle est realisée sur PCB

**Comment by f4bjh** [comment]
>Pour J4 et J7 ne peut on pas mettre des points info comme J2 , J3 ou J6
>si je me trompe pas c'est la que vas venir "la tension" mesurée aux bornes de la ligne coaxial


Non. Car ce n'est pas "la tension à mesurer" qui arrive sur J4 ou J7. Mais les signaux HF transmis et réfléchis


>pour j4 et j7 je peut on pas les mettre de chaque cote du PCB toujours par rapport a la ligne coaxiale ??
Oui ça c'est possible

>ne doit on pas fixer une taille maxi du PCB ?? afin d'optimiser au mieux l'emplacement des composants ? peut être a voir en >fonction de la ligne coaxial si elle est realisée sur PCB

Oui.
